Classes/Service/ConfigurationService.php

```diff
@@ -406,6 +406,15 @@ protected function buildWizardTabItem($tabId, $id, $form, $templateFileIdentity)
 				$iconRegistry->registerIcon($iconIdentifier, BitmapIconProvider::class, array('source' => $icon));
 			}
 		}
+		$defaultValues = array();
+		if ($form->hasOption(Form::OPTION_DEFAULT_VALUES)) {
+			foreach ($form->getOption(Form::OPTION_DEFAULT_VALUES) as $key => $value) {
+				if (TRUE === is_array($value)) {
+					$value = http_build_query($value);
```

Comment:
$value after this will be a string like `foo=bar&baz=xyz` which means that below, the string would become `dingo=foo=bar&baz=xyz` if $key is `dingo`. I think we need to use `http_build_query` on the entire array returned by `$form->getOption(Form::OPTION_DEFAULT_VALUES)` (it should also work recursively when array contains other arrays, associative or otherwise).